### PR TITLE
Configure codestyle settings for Intellij IDEA

### PR DESCRIPTION
**Pull request by zarechenskiy**

### CODE REVIEW

**Comment by sandwwraith** [comment]
It's weird, but animalsniffer issue is stably reproduced on this branch (although looking unrelated). Can you rebase it on latest dev, please?

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,8 @@
 # IntelliJ files
 **/.idea/*
 !/.idea/vcs.xml
+!/.idea/codeStyles
+!/.idea/copyright
```
Comment (shanshin) [inline .gitignore:7]: Should this files also be commited?

**Comment by shanshin** [comment]
> but animalsniffer issue is stably reproduced on this branch

Yes, it's unrelated, fixed by 24c60287

File: .gitignore (hunk 1 above)
Comment (sandwwraith) [reply .gitignore:7]: I think these are folders with profiles

**Pull request merged**